Packs/CommonScripts/Scripts/StixCreator/StixCreator_test.py

```diff
@@ -85,3 +117,102 @@ def test_stixCreator_with_indicators(mocker, indicators, stix_type):
 def test_guess_indicator_type(k, v, exp):
     a = guess_indicator_type(k, v)
     assert a == exp
+
+
+xsoar_indicator_1 = {'expirationStatus': 'active',
+                     'firstSeen': '2023-04-19T17:43:07+03:00',
+                     'indicator_type': 'Account',
+                     'lastSeen': '2023-04-19T17:43:07+03:00',
+                     'score': 'Unknown',
+                     'timestamp': '2023-04-19T17:43:07+03:00',
+                     'value': 'test@test.com'}
+stix_type_1 = "user-account"
+value_1 = 'test@test.com'
+expected_stix_id_1 = "user-account--783b9e67-d7b0-58f3-b566-58ac7881a3bc"
+
+xsoar_indicator_2 = {'expirationStatus': 'active',
+                     'firstSeen': '2023-04-20T10:20:04+03:00',
+                     'indicator_type': 'File',
+                     'lastSeen': '2023-04-20T10:20:04+03:00',
+                     'score': 'Unknown', 'sourceBrands': 'VirusTotal',
+                     'sourceInstances': 'VirusTotal',
+                     'timestamp': '2023-04-20T10:20:04+03:00',
+                     'value': '701393b3b8e6ae6e70effcda7598a8cf92d0adb1aaeb5aa91c73004519644801'}
+stix_type_2 = "file"
+value_2 = '701393b3b8e6ae6e70effcda7598a8cf92d0adb1aaeb5aa91c73004519644801'
+expected_stix_id_2 = "file--3e26aab3-dfc3-57c5-8fe2-45cfde8fe7c8"
+
+xsoar_indicator_3 = {'expirationStatus': 'active',
+                     'firstSeen': '2023-04-18T12:17:38+03:00',
+                     'indicator_type': 'IP',
+                     'lastSeen': '2023-04-18T12:17:38+03:00',
+                     'score': 'Unknown',
+                     'timestamp': '2023-04-18T12:17:38+03:00',
+                     'value': '8.8.8.8'}
+stix_type_3 = "ipv4-addr"
+value_3 = '8.8.8.8'
+expected_stix_id_3 = "ipv4-addr--2f689bf9-0ff2-545f-aa61-e495eb8cecc7"
+
+test_test_create_sco_stix_uuid_params = [(xsoar_indicator_1, stix_type_1, value_1, expected_stix_id_1),
+                                         (xsoar_indicator_2, stix_type_2, value_2, expected_stix_id_2),
+                                         (xsoar_indicator_3, stix_type_3, value_3, expected_stix_id_3)]
+
+
+@pytest.mark.parametrize('xsoar_indicator, stix_type, value, expected_stix_id', test_test_create_sco_stix_uuid_params)
+def test_create_sco_stix_uuid(xsoar_indicator, stix_type, value, expected_stix_id):
+    """
+    Given:
+        - An indicator from XSOAR, it's stix type and the value of the indicator.
+    When:
+        - Creating a SCO inducator and calling create_sco_stix_uuid
+    Then:
+     - Return the indicator id.
+    """
```

Comment:
Let's add cases to the 'Given' and 'Then' parts that describe the different inputs and different expectations we are checking. For example:
```suggestion
    Given:
        - Case 1: An XSOAR indicator of type 'Account', with a stix type of 'user-account' and a value of 'test@test.com'.
        .....
    When:
        - Creating a SCO inducator and calling create_sco_stix_uuid
    Then:
     - Assert the ID looks like XXX
     .....
    """
```